Sources_v3/StreamChatUI/Common Views/OnlineIndicatorView.swift

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class OnlineIndicatorView: UIView {
```

Comment:
features:
-> is circle
-> has the inner circle

-> backgroundColor (default)
-> indicatorColor